nil pointer dereference in firewall.go

After setting an API Key with CF_API_KEY and CF_API_EMAIL, I can successfully run `flarectl firewall rules list` and get and empty list as expected:

```
  ID | VALUE | SCOPE | MODE | NOTES
-----+-------+-------+------+--------
```

However, `flarectl firewall rules create --zone crud.net --mode=block --value='{"target":"ip","value","198.51.100.4"}'` panics with a nil pointer dereference:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x7cf7be]

goroutine 1 [running]:
main.firewallAccessRuleCreate(0xc000072f40, 0x6, 0xa)
        /home/jsha/go/src/github.com/cloudflare/cloudflare-go/cmd/flarectl/firewall.go:123 +0x43e
github.com/urfave/cli/v2.(*Command).Run(0xc0001a6c60, 0xc000072d00, 0x0, 0x0)
        /home/jsha/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/command.go:163 +0x4ed
github.com/urfave/cli/v2.(*App).RunAsSubcommand(0xc00010b6c0, 0xc000072b40, 0x0, 0x0)
        /home/jsha/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:434 +0xa9f
github.com/urfave/cli/v2.(*Command).startApp(0xc0001a6a20, 0xc000072b40, 0x8bc53a, 0x7fffee7fb30d)
        /home/jsha/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/command.go:278 +0x69b
github.com/urfave/cli/v2.(*Command).Run(0xc0001a6a20, 0xc000072b40, 0x0, 0x0)
        /home/jsha/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/command.go:94 +0x98d
github.com/urfave/cli/v2.(*App).RunAsSubcommand(0xc00010b520, 0xc000072a80, 0x0, 0x0)
        /home/jsha/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:434 +0xa9f
github.com/urfave/cli/v2.(*Command).startApp(0xc0001a6900, 0xc000072a80, 0x8bed12, 0x7fffee7fb304)
        /home/jsha/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/command.go:278 +0x69b
github.com/urfave/cli/v2.(*Command).Run(0xc0001a6900, 0xc000072a80, 0x0, 0x0)
        /home/jsha/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/command.go:94 +0x98d
github.com/urfave/cli/v2.(*App).RunContext(0xc00010b380, 0x945c20, 0xc000016108, 0xc000012080, 0x8, 0x8, 0x0, 0x0)
        /home/jsha/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:313 +0x81f
github.com/urfave/cli/v2.(*App).Run(...)
        /home/jsha/go/pkg/mod/github.com/urfave/cli/v2@v2.3.0/app.go:224
main.main()
        /home/jsha/go/src/github.com/cloudflare/cloudflare-go/cmd/flarectl/flarectl.go:686 +0x4870
```

## Environment

Include as many relevant details about the environment you experienced
the bug in:

* Version used (e.g. git sha or specific tag): 9f836c42
* Environment name and version (e.g. Go 1.11, MacOS 10.14.1): Ubuntu 20.04, Go 1.15.

Note this is a different crash than the one I reported in #584 .

--- a/cmd/flarectl/firewall.go
+++ b/cmd/flarectl/firewall.go
@@ -3,6 +3,7 @@ package main
 import (
 	"fmt"
 	"net"
+	"os"
 	"strconv"
 
 	"github.com/pkg/errors"
@@ -104,27 +105,29 @@ func firewallAccessRuleCreate(c *cli.Context) error {
 	}
 
 	var (
-		rules       []cloudflare.AccessRule
-		errCreating = "error creating firewall access rule"
+		rules []cloudflare.AccessRule
 	)
 
 	switch {
 	case accountID != "":
 		resp, err := api.CreateAccountAccessRule(accountID, rule)
 		if err != nil {
-			errors.Wrap(err, errCreating)
+			fmt.Fprintln(os.Stderr, "error creating account access rule: ", err)
+			return err
 		}
 		rules = append(rules, resp.Result)
 	case zoneID != "":
 		resp, err := api.CreateZoneAccessRule(zoneID, rule)
 		if err != nil {
-			errors.Wrap(err, errCreating)
+			fmt.Fprintln(os.Stderr, "error creating zone access rule: ", err)
+			return err
 		}
 		rules = append(rules, resp.Result)
 	default:
 		resp, err := api.CreateUserAccessRule(rule)
 		if err != nil {
-			errors.Wrap(err, errCreating)
+			fmt.Fprintln(os.Stderr, "error creating user access rule: ", err)
+			return err
 		}
 		rules = append(rules, resp.Result)
 

